modules/ROOT/pages/design-create-publish-api-raml-editor.adoc

```diff
@@ -1,23 +1,26 @@
 = Create an API Specification with the Text Editor
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
 
 You can use API Designer's text editor to create API specifications directly in RAML 0.8, RAML 1.0, or OAS 2.0 or 3.0 (either in JSON or YAML).
 
 == Before You Begin
+
 Within Anypoint Platform, your user ID must be assigned the _Design Center Developer_ permission.
 
 == Procedure
 
-. Open Design Center from the homepage of Anypoint Platform.
-. Create an API-specification project that you can work in.
- .. On the *Projects* page in Design Center, click *Create* and then click *Create API Specification*.
- .. In the *New Specification* section of the dialog that is opened, name your project. You can change the name (but not the extension) later, if you want to.
- .. Select *Text editor*.
- .. Choose whether the stub specification file created in the project should be in RAML or OAS. The default is to create a RAML 1.0 file; however, you can change the file to RAML 0.8 or OAS after the project is created. In fact, projects for creating API specifications can contain a mix of RAML and OAS files.
+. On the *Projects* page in Design Center, click *Create new*.
+. Select *New API Spec*.
+. In the *New API Spec* dialog, name your project.
++
+You can change the name (but not the extension) later, if you want to.
```

Comment:
I don't really recall what we're meaning with extension

Reply:
I took out "but not the extension"

Reply:
did you mean file extension (.yml)? and it is still there?